torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -759,6 +764,7 @@ def pad_bounding_box(
 
     bounding_box = bounding_box.clone()
 
+    # TODO: avoid repeated indexing and perform concurrently. `output.add_(torch.tensor([left, top, left, top]))` is 3x faster
```

Comment:
A quick check of:
```
def single(inpt, left, top):
    output = inpt.clone()
    output[..., 0] += left
    output[..., 1] += top
    output[..., 2] += left
    output[..., 3] += top

    return output


def concurrent(inpt, left, top):
    output = inpt.clone()
    output.add_(torch.tensor([left, top, left, top]))

    return output
```

Shows:
```
[-------- Pad cpu torch.float32 --------]
               |  single   |  concurrent 
1 threads: ------------------------------
      (16, 4)  |     60    |       20    
6 threads: ------------------------------
      (16, 4)  |     58    |       15    

Times are in microseconds (us).
```

There might be multiple opportunities for speed up like this.